.github/PULL_REQUEST_TEMPLATE

```diff
@@ -0,0 +1,14 @@
+### Milestone (Odoo version)
```

Comment:
I see this file no real advantage, as we have multiple PR types:

* New modules
* Feature fixes
* CIs fixes
* Maintenance PRs
* ...